test/extended/util/framework.go

```diff
@@ -236,127 +236,152 @@ func processScanError(log string) error {
 // WaitForOpenShiftNamespaceImageStreams waits for the standard set of imagestreams to be imported
 func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 	ctx := context.Background()
+	langs := []string{"ruby", "nodejs", "perl", "php", "python", "mysql", "postgresql", "jenkins"}
```

Comment:
I mentioned this in the slack threads around this, but allow me to echo here.  And perhaps item asked in the various slack threads was not the root cause of the CI failure(s).

But with that preamable :-) 

The only thing the install payload imagestreams defined at https://github.com/openshift/cluster-samples-operator/blob/master/manifests/08-openshift-imagestreams.yaml have to do with samples operator is that Clayton decided to put the imagestream defs in the manifest dir of the samples operator repo.

No creation/update, no monitoring, no tracking, no management, no retries from samples operator for those.

The CVO in fact creates/updates the items in https://github.com/openshift/cluster-samples-operator/blob/master/manifests/08-openshift-imagestreams.yaml

One thing I'll highlight:  they all have the `scheduled` bit turned on.  i.e. https://github.com/openshift/cluster-samples-operator/blob/master/manifests/08-openshift-imagestreams.yaml#L89

That means if the initial import fails, the imagestream related controllers in the OCM will try again in 10 to 15 minutes :-)

Reply:
Can this list also include the imagestream - `tools` that provides `openshift/tools` image  which is used extensively throughout the tests referred to as `ShellImage()`. 
Also, All calls to `CreateExecPodOrFail` use this image.

Reply:
no it cannot @sthaha , at least not in this function, the tools imagestream will not show up in the samples operator object that is inspected below and the initiation of retries that is done if the import fails, since it is not being tracked at 
per my comment at https://github.com/openshift/origin/pull/26472/files#r711270195

but I will note that all the in payload imagesterams have scheduled import turned on, so the imagestream controller will retry every 15 minutes if there is a failure

more restructure would be needed if you wanted to expedite retry on things like tools for imagestreams not managed by samples operator